docs/setup/keadm.md

```diff
@@ -95,6 +95,49 @@ Host has mosquit+ already installed and running. Hence skipping the installation
 KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
 ```
 
+### Enable `kubectl logs` Feature
+1. Make sure you can find the kubernetes `ca.crt` and `ca.key` files. If you set up your kubernetes cluster by `kubeadm` , those files will be in `/etc/kubernetes/pki/` dir.
```

Comment:
What are the `ca.crt` and `ca.key` used  for here? @YaozhongZhang

Reply:
@fisherxu They are used to sign the certificates used by CloudStream.

Reply:
Okey